### PR TITLE
Crw-2822 update CRW variables to DS

### PR DESCRIPTION
**Pull request by SDawley**
### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-2822

### CODE REVIEW

File: build/scripts/build.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ PUBLISH_ARTIFACTS_TO_GITHUB=0
 PUBLISH_ARTIFACTS_TO_RCM=0
 
 # for publishing to RCM only
-DESTHOST="crw-build/codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com@rcm-guest.app.eng.bos.redhat.com"
+DESTHOST="crw-build@rcm-guest.app.eng.bos.redhat.com"
```
Comment (nickboldt) [inline build/scripts/build.sh:31]: are you sure about this change?

File: build/scripts/build.sh [hunk 2]
```diff
@@ -149,16 +149,16 @@ if [[ $DO_SYNC -eq 1 ]]; then
...
         for d in ${FILES}; do
-            sed -i ${d} -r -e "s#registry.redhat.io/devspaces/(.+):(.+)#registry.redhat.io/devspaces/\1:${CRW_VERSION}#g"
+            sed -i ${d} -r -e "s#registry.redhat.io/devspaces/(.+):(.+)#registry.redhat.io/devspaces/\1:${DS_VERSION}#g"
         done
     popd >/dev/null
 
     pushd $DSC_DIR >/dev/null
         ./build/scripts/sync.sh -b ${MIDSTM_BRANCH} -s ${SOURCE_DIR} -t ${DSC_DIR} \
-            --crw-version ${CRW_VERSION}
+            --crw-version ${DS_VERSION}
```
Comment (nickboldt) [inline build/scripts/build.sh:161]: see https://gitlab.cee.redhat.com/codeready-workspaces/crw-jenkins/-/merge_requests/1157/diffs#9c7953f5487b41b481419fc1c87a027f2f2291e9_54_53

File: build/scripts/sync.sh [hunk 3]
```diff
@@ -35,7 +35,7 @@ while [[ "$#" -gt 0 ]]; do
 	'-t') TARGETDIR="$2"; TARGETDIR="${TARGETDIR%/}"; shift 1;;
 	'--help'|'-h') usage;;
 	# optional tag overrides
-	'--crw-version') CRW_VERSION="$2"; DEFAULT_TAG="$2"; shift 1;;
+	'--crw-version') DS_VERSION="$2"; DEFAULT_TAG="$2"; shift 1;;
```
Comment (nickboldt) [inline build/scripts/sync.sh:38]: see https://gitlab.cee.redhat.com/codeready-workspaces/crw-jenkins/-/merge_requests/1157/diffs#9c7953f5487b41b481419fc1c87a027f2f2291e9_54_53

File: build/scripts/build.sh [hunk 4]
```diff
@@ -84,11 +84,11 @@ while [[ "$#" -gt 0 ]]; do
 	# paths to use for input and ouput
 	'-s') SOURCE_DIR="$2"; SOURCE_DIR="${SOURCE_DIR%/}"; shift 1;;
 	'-t') DSC_DIR="$2"; DSC_DIR="${DSC_DIR%/}"; shift 1;;
-	'-i') CRWIMG_DIR="$2"; CRWIMG_DIR="${CRWIMG_DIR%/}"; shift 1;;
+	'-i'DSIMG_DIR="$2"; DSIMG_DIR="${DSIMG_DIR%/}"; shift 1;;
 	'--help'|'-h') usageSegKey;;
 	# optional tag overrides
     '--suffix') versionSuffix="$2"; shift 1;;
-	'--crw-version') CRW_VERSION="$2"; DEFAULT_TAG="$2"; shift 1;;
+	'--crw-version') DS_VERSION="$2"; DEFAULT_TAG="$2"; shift 1;;
```
[the remark this answers is not in the document]
Comment (SDawley) [reply build/scripts/build.sh:91]: Was planning on doing that as part of https://issues.redhat.com/browse/CRW-3071

[the remark this answers is not in the document]
Comment (nickboldt) [reply build/scripts/build.sh:91]: OK, then I've reverted the change to the job so it's not broken in the interim. https://gitlab.cee.redhat.com/codeready-workspaces/crw-jenkins/-/merge_requests/1197/diffs?commit_id=5bc8e2e836339bcc7ec49204b1feaa0ef1400ae7

**Pull request merged**